### PR TITLE
chore(canary): remove unneeded files under `canary/`

### PR DESCRIPTION
**Pull request by wlee221**
#### Description of changes

`canary/` folder had some outdated files that were hanging in parent folders. They were not used in any of the apps, so this PR removes it.

#### Description of how you validated changes

Ran `yarn install` and `yarn build` successfully on `canary`.

#### Checklist

- [x] PR description included
- [x] `yarn test` passes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: canary/apps/angular/.browserslistrc [hunk 1]
```diff
@@ -1,17 +0,0 @@
-# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
```
Comment (wlee221) [inline canary/apps/angular/.browserslistrc:1]: This `.rc` files are already under `canary/apps/angular/angularcli`, and doesn't need to be duplicated here. Same with the other files.

File: canary/apps/react/App.js [hunk 2]
```diff
@@ -1,21 +0,0 @@
...
-    <AmplifyProvider colorMode="dark">
-      <Card>
-        <Heading>Amplify Sample App</Heading>
-        <Button variation="primary">Click me!</Button>
-      </Card>
-    </AmplifyProvider>
-  );
-}
-
-export default withAuthenticator(App);
```
Comment (ErikCH) [inline canary/apps/react/App.js:21]: Don't we need this to test the Authenticator?

File: canary/apps/react/App.js [hunk 3]
```diff
@@ -1,21 +0,0 @@
-import {
```
Comment (reesscot) [inline canary/apps/react/App.js:1]: 🤦‍♂️

**Review by reesscot — APPROVED** [review]

**Review by ErikCH — APPROVED** [review]

File: canary/apps/react/App.js (hunk 3 above)
Comment (ErikCH) [reply canary/apps/react/App.js:1]: For a minute I thought this was in the `react/src` not just in the react subfolder

Comment (ErikCH) [reply canary/apps/react/App.js:1]: LGTM

**Pull request merged**